internal/ceb/server.go

```diff
@@ -153,8 +159,40 @@ func (ceb *CEB) dialServer(ctx context.Context, cfg *config, isRetry bool) error
 			return err
 		}
 
+		token := resp.Token
+
+		var perRPC credentials.PerRPCCredentials = staticToken(token)
```

Comment:
Huh! Never seen this before...any reason why this could be simplified to `perRPC := staticToken(token)`?

Reply:
actually no! The reason is I want `perRPC` to be the interface type, and if I just do `perRPC := staticToken(token)` then it will be `staticToken` type. The reason being I want to reassign it to a different implementor of `PerRPCCredentials` in the code below this.